pilot/pkg/kube/inject/inject_test.go

```diff
@@ -37,6 +41,12 @@ const (
 	// Tag name should be kept in sync with value in
 	// platform/kube/inject/refresh.sh
 	unitTestTag = "unittest"
+
+	statusReplacement = "sidecar.istio.io/status: '{\"version\":\"\","
```

Comment:
nit: use `` to avoid the \", here and below.

Reply:
I don't want to change the type of quotes used in the files.  They're currently using `"`.